src/firefly/js/ui/panel/CollapsiblePanel.jsx

```diff
@@ -25,85 +24,87 @@ export const CollapseHeaderCorner = {
+        setIsOpenState(newIsOpen);
 
         if (onToggle) {
-            onToggle(isOpen);
+            onToggle(newIsOpen);
         }
     }
 
-    getContentHeight() {
-        return this.state.isOpen ? {height: 'initial'} : {height: 0};
+
+    const contentBorderClassName = ['', ' CollapsiblePanel__Content-oneborder',
+        ' CollapsiblePanel__Content-twoborder', ' CollapsiblePanel__Content-threeborder'];
+    let headerClassName = 'CollapsiblePanel__Header';
+    let contentClassName = 'CollapsiblePanel__Content';
+    let headerCorner = '';
+    ['TopLeft', 'TopRight', 'BottomRight', 'BottomLeft'].forEach((corner) => {
+        CollapseHeaderCorner[corner]&headerRoundCorner ?
+            headerCorner += ' 0.5em' : headerCorner += ' 0em';
+    });
+
+    headerStyle = { ...headerStyle, 'borderRadius': headerCorner};
+
+    if (isOpenState) {
+        headerClassName += ' CollapsiblePanel__Header--is-open';
+        contentClassName += contentBorderClassName[borderStyle];
     }
 
-    render () {
-        const {header, headerRoundCorner, borderStyle, wrapperStyle, children} = this.props;
-        var {headerStyle, contentStyle} = this.props;
-        const contentBorderClassName = ['', ' CollapsiblePanel__Content-oneborder',
-                              ' CollapsiblePanel__Content-twoborder', ' CollapsiblePanel__Content-threeborder'];
-        let headerClassName = 'CollapsiblePanel__Header';
-        let contentClassName = 'CollapsiblePanel__Content';
-        let headerCorner = '';
-
-       ['TopLeft', 'TopRight', 'BottomRight', 'BottomLeft'].forEach((corner) => {
-            CollapseHeaderCorner[corner]&headerRoundCorner ?
-                                            headerCorner += ' 0.5em' : headerCorner += ' 0em';
-       });
-
-        headerStyle = Object.assign({}, headerStyle, {'borderRadius': headerCorner});
-
-        if (this.state.isOpen) {
-            headerClassName += ' CollapsiblePanel__Header--is-open';
-            contentClassName += contentBorderClassName[borderStyle];
-        }
+    function getContentHeight() {
+        return isOpenState ? {height: 'initial'} : {height: 0};
+    }
+    contentStyle = {display: 'block', ...contentStyle, ...getContentHeight()};
+
+    const headerContent = isFunction(header) ? header() : header;
+
 
-        contentStyle = Object.assign({display: 'block'}, contentStyle, this.getContentHeight());
-        const headerContent = isFunction(header) ? header() : header;
-
-        return (
-            <div style={{display: 'flex', flexDirection: 'column', ...wrapperStyle}}>
-                <div style={headerStyle} onClick={this.handleClick} className={headerClassName}>
-                    {headerContent}
-                </div>
-                <div style={contentStyle} className={contentClassName}>
-                    {children}
-                </div>
+    return (
+        <div style={{display: 'flex', flexDirection: 'column', ...wrapperStyle}}>
```

Comment:
We made a mistake a few years make by using the term `wrapperStyle`.  Change prop `wrapperStyle` to `style`. And change any code that users it.

To do this you will need use intelliJ to do a find usages on both `CollapsiblePanel` and `FieldGroupCollapsible`. I think there are only two or three files that actually use `wrapperStyle`